swift/example_code/bedrock-runtime/models/amazon-nova/amazon-nova-text/Sources/Converse/main.swift

```diff
@@ -0,0 +1,57 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+// 
+// snippet-start:[swift.example_code.bedrock-runtime.Converse_AmazonNovaText]
+// An example demonstrating how to use the Conversation API to send a text message to Amazon Nova.
```

Comment:
Please try to wrap lines to around 80-90 characters, more or less, when
possible. This makes them look better when embedded in the developer guide.